### DISCUSSION TITLE
Could pandapower load Matpower .m file as the input files?

### DISCUSSION THREAD

**Discussion by XinFang88** [e2nIEE/pandapower#1333] [Q&A]
Hi,

I'm new to this pandapower package. I have a question about the Pandapower. Could pandapower load Matpower .m file as the input files? If so, is there any example about it?

Best,
Xin

**Comment by rbolgaryn** [comment]
Hi @xfangnrel 

you can use the matpower converters to load matpower files:

https://github.com/e2nIEE/pandapower/tree/develop/pandapower/converter/matpower

as described in the documentation:


https://pandapower.readthedocs.io/en/v2.6.0/converter/matpower.html


Roman

**Comment by SteffenMeinecke** [comment]
For loading data from `.m` files (not `.mat` files), please follow the process of issue https://github.com/e2nIEE/pandapower/issues/1556.
